doc/manual/meson.build

```diff
@@ -260,7 +263,7 @@ nix3_manpages = [
   'nix3-repl',
   'nix3-run',
   'nix3-search',
-  #'nix3-shell',
```

Comment:
No man pages for aliases; a meson independent problem.

- Tracked in https://github.com/NixOS/nix/issues/11430